docs/generated/oc_by_example_content.adoc

```diff
@@ -1695,6 +1695,10 @@ Expose a replicated application as a service or route
   # Create a route and specify a hostname
   oc expose service nginx --hostname=www.example.com
   
+  # Create a route with wildcard
+  oc expose service nginx --hostname=x.example.com --wildcard=Subdomain
+  **NOTE: The subdomain wildcard policy works on the first level subdomain.
```

Comment:
This isn't clear to me... perhaps saying:
This would be equivalent to *.example.com.  NOTE: only hosts are matched by the wildcard, subdomains would not be included.